.changeset/cool-elephants-leave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/auth": patch
+---
+
+Add missing field to firebase claim in token result typing
```

Comment:
Should Firebase be capped here, or is it `firebase` a literal?

Reply:
Firebase literal, should I put it in backticks?

Reply:
Updated for backticks